daisy_workflows/build-publish/kokoro/kokoro_signing.publish.json

```diff
@@ -0,0 +1,29 @@
+{
+  "Name": "kokoro-signing",
+  "WorkProject": "gce-image-builder",
+  "ComputeEndpoint": "https://www.googleapis.com/compute/v1/projects/",
+  "DeleteAfter": "24h*30*6",
+  {{if eq .environment "test" -}}
```

Comment:
Will we ever publish this image to more than 1 project? If not this entire if/else approach can be removed.